include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl.h

```diff
@@ -806,14 +806,16 @@ struct __red_by_seg_op
     operator()(const _Tup1& __lhs_tup, const _Tup2& __rhs_tup) const
     {
         using std::get;
+        using _OpOutType = decltype(__binary_op(get<1>(__lhs_tup), get<1>(__rhs_tup)));
```

Comment:
```suggestion
        using _OpReturnType = decltype(__binary_op(get<1>(__lhs_tup), get<1>(__rhs_tup)));
```
I think `_OpReturnType` is a slightly clearer name but either works in my opinion.

Reply:
I agree it's better and made this change.